vms/platformvm/txs/executor/proposal_tx_executor.go

```diff
@@ -270,34 +269,17 @@ func (e *ProposalTxExecutor) AdvanceTimeTx(tx *txs.AdvanceTimeTx) error {
 		)
 	}
 
-	parentChainTime := e.OnCommitState.GetTimestamp()
-	if !newChainTime.After(parentChainTime) {
```

Comment:
This used to verify that `parentChainTime < newChainTime`. Now the code verifies that `parentChainTime <= newChainTime`. Technically this verification could be entirely removed as `AdvanceTimeTx`s are now only verified during bootstrapping.

Reply:
In bootstrapping, everything we are verifying has been attested to have been accepted by at least 51% of the network. So, we know that a majority of the network was able to verify and accept this operation. So, actually verifying that it is valid locally is really not needed.

During normal consensus, we need to verify blocks that haven't been accepted yet.. So it's important that we verify every rule condition.

In this case, this transaction was disabled during the Banff upgrade, so any new block containing this transaction would have already errored out from the `e.Config.UpgradeConfig.IsBanffActivated(newChainTime)` check above here... So we know that this code is never hit during normal operations.

Reply:
This was the minimal diff... It could be removed